src/client/components/pages/entities/creator.js

```diff
@@ -121,6 +121,15 @@ function CreatorDisplayPage({entity, identifierTypes}) {
 				identifierTypes={identifierTypes}
 				urlPrefix={urlPrefix}
 			/>
+			<Button
+				bsStyle="success"
+				className="margin-top-d15"
+				href={`/work/create?${
+					entity.type.toLowerCase()}=${entity.bbid}`}
+			>
+				<Icon name="plus"/>
+				{'  Add Work'}
```

Comment:
This is unnecessary. You should just be able to have a space between the Icon element and regular text like this:
`<Icon name="plus"/> Add Work`.

If you want more space betwen the icon and the work, you can add `className='margin-right-0-5'`to the Icon element, but I'm not sure that's necessary.